main.scss

```diff
@@ -4,18 +4,45 @@
 @import 'o-typography/main';
 
 @import './src/scss/color-use-cases';
-@import './src/scss/deprecated';
 @import './src/scss/brand';
 @import './src/scss/mixins';
 @import './src/scss/variables';
 
-@import './src/scss/shared/mixins';
-@import './src/scss/shared/placeholders';
-@import './src/scss/shared/status';
+@import './src/scss/base';
+@import './src/scss/state';
 
-@import './src/scss/alert/main';
-@import './src/scss/notice/main';
-@import './src/scss/action/main';
+/// Output all oMessage features
+/// @param {list} $types [('alert', 'notice', 'action', 'inner'))]
+/// @param {list} $status [('error', 'success', 'inform', 'neutral', 'inform-inverse', 'warning', 'warning-light')]
+
+@mixin oMessage (
+	$opts: ('action', 'alert', 'notice', 'inner'),
```

Comment:
Is this meant to be `$types`?

_Edit: Or maybe the SassDocs should be `$opts`?_

Reply:
SassDocs should be opts 😬